Packs/AWS-SNS/Integrations/AWSSNS/AWSSNS.py

```diff
@@ -0,0 +1,407 @@
+            })
+            sts_response = sts_client.assume_role(**kwargs)
+            client = boto3.client(
+                service_name=service,
+                region_name=self.aws_default_region,
+                aws_access_key_id=sts_response['Credentials']['AccessKeyId'],
+                aws_secret_access_key=sts_response['Credentials']['SecretAccessKey'],
+                aws_session_token=sts_response['Credentials']['SessionToken'],
+                verify=self.verify_certificate,
+                config=self.config
+            )
+        elif self.aws_access_key_id and not self.aws_role_arn:  # login with access key id
+            client = boto3.client(
+                service_name=service,
+                region_name=region if region else self.aws_default_region,
+                aws_access_key_id=self.aws_access_key_id,
+                aws_secret_access_key=self.aws_secret_access_key,
+                verify=self.verify_certificate,
+                config=self.config
+            )
+        elif self.aws_session_token and not self.aws_role_arn:  # login with session token
+            client = boto3.client(
+                service_name=service,
+                region_name=region if region else self.aws_default_region,
+                aws_access_key_id=self.aws_access_key_id,
+                aws_secret_access_key=self.aws_secret_access_key,
+                aws_session_token=self.aws_session_token,
+                verify=self.verify_certificate,
+                config=self.config
+            )
+        else:  # login with default permissions, permissions pulled from the ec2 metadata
+            client = boto3.client(service_name=service,
+                                  region_name=region if region else self.aws_default_region)
+
+        return client
+
+    @staticmethod
+    def get_timeout(timeout):
+        if not timeout:
+            timeout = "60,10"  # default values
+        try:
+
+            if isinstance(timeout, int):
+                read_timeout = timeout
+                connect_timeout = 10
+
+            else:
+                timeout_vals = timeout.split(',')
+                read_timeout = int(timeout_vals[0])
+                # the default connect timeout is 10
+                connect_timeout = 10 if len(timeout_vals) == 1 else int(timeout_vals[1])
+
+        except ValueError:
+            raise DemistoException("You can specify just the read timeout (for example 60) or also the connect "
+                                   "timeout followed after a comma (for example 60,10). If a connect timeout is not "
+                                   "specified, a default of 10 second will be used.")
+        return read_timeout, connect_timeout
+
+
+register_module_line('AWSApiModule', 'end', __line__(), wrapper=1)
+### END GENERATED CODE ###
+
+if __name__ in ('__main__', '__builtin__', 'builtins'):
+    main()
+
+register_module_line('AWS - SNS', 'end', __line__())
```

Comment:
please remove this line
it will generated automatically

Reply:
@ilappe Done